Classes/Replicator/NodeReplicator.php

```diff
@@ -26,12 +27,18 @@ class NodeReplicator
      */
     protected $logger;
 
+    /**
+     * @Flow\Inject
+     * @var PersistenceManagerInterface
+     */
+    protected $persistenceManager;
```

Comment:
Seems not be needed?

Reply:
Oh right, i first thought i need to persist the change 🙈 
Will be removed...